### PR TITLE
hardcode js run code

### PR DESCRIPTION
**Pull request by yviansu**

### CODE REVIEW

File: tests/benchmark/array_access_i32.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@
  * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
  */
 
+type i32 = number;
```
Comment (xujuntwt95329) [inline tests/benchmark/array_access_i32.ts:6]: Do we require app to write this line?

Comment (yviansu) [reply tests/benchmark/array_access_i32.ts:6]: For compiling ts to wasm, we don't need this line, but we also need to compile ts to js, in this situation, we need to set i32 as a resonable type.

File: tests/benchmark/run_benchmark_on_chrome.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+ * Copyright (C) 2023 Intel Corporation.  All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+import { importObject, setWasmMemory } from '../../tools/validate/run_module/import_object.js';
+
+const compile_output_dir = './compile_output/';
+
+export function run_wasm_file(fileName, warmupTimes, cell) {    
+    fetch(compile_output_dir.concat(fileName, '.wasm'))
```
Comment (xujuntwt95329) [inline tests/benchmark/run_benchmark_on_chrome.js:11]:
```suggestion
    return fetch(compile_output_dir.concat(fileName, '.wasm'))
```

File: tests/benchmark/run_benchmark_on_chrome.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2023 Intel Corporation.  All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+import { importObject, setWasmMemory } from '../../tools/validate/run_module/import_object.js';
+
+const compile_output_dir = './compile_output/';
+
+export function run_wasm_file(fileName, warmupTimes, cell) {
```
Comment (xujuntwt95329) [inline tests/benchmark/run_benchmark_on_chrome.js:10]:
```suggestion
export async function run_wasm_file(fileName, warmupTimes, cell) {    
```

File: tests/benchmark/run_benchmark_on_chrome.html [hunk 4]
```diff
@@ -0,0 +1,453 @@
...
+        // merkletrees_time();
+        nbody_class_time();
+        nbody_interface_time();
+        quicksort_time();
+        quicksort_float_time();
+        spectral_norm_time();
+        spectral_norm_i32_time();
+
+        /* update wasm run times */
+        run_wasm_file('any_basic_type_access', warmupTimes);
```
Comment (xujuntwt95329) [inline tests/benchmark/run_benchmark_on_chrome.html:428]:
```suggestion
        await run_wasm_file('any_basic_type_access', warmupTimes);
```

Same for below lines

**Pull request merged**